packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingForm.js

```diff
@@ -30,16 +30,22 @@ const ShippingForm = props => {
         }
     });
     const {
+        errorMessage,
         handleOnSubmit,
         handleZipChange,
         isSetShippingLoading
     } = talonProps;
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
+    const errorElement = errorMessage ? (
+        <span className={classes.error}>{errorMessage}</span>
```

Comment:
I might have liked to see a `FormError` component for form-level errors, since I imagine the presentation will be nearly identical for all of them. Such a component could render `null` when `props.message` was empty, too, cutting down on a lot of the boilerplate.

Reply:
Added `FormError` component in 57ab1a7, and changed all usages in this PR to use it (where it didn't dramatically alter the UX).

I thought we had discussed deferring a dedicated component to a followup Story, but I've gone ahead and implemented one in this scope.

Reply:
Thanks. I apologize if we backtracked on an agreement. It's not the duplication of code that bothered me; I just saw that we were adding new classnames to a lot of form components as a result of the new error elements, and thought it'd be nice to avoid that churn. Glad to have a component now.